hardhat.config.ts

```diff
@@ -113,8 +113,8 @@ const config: HardhatUserConfig = {
     thresholdCouncil: {
       mainnet: "0x9F6e831c8F8939DC0C830C6e492e7cEf4f9C2F5f",
     },
-    keepDeployer: {
-      default: 0,
+    keepRegistryKeeper: {
```

Comment:
Maybe `keepRegistryOwner` or `keepRegistryGuvnor`? Keeper of keep registry sounds weird but if you don't think the alternatives are any better, I can also merge as-is.

Reply:
I agree it doesn't sound good, but I wanted to match the role that is defined in `KeepRegistry` that is named `registryKeeper`.